src/components/vm/vmDetailsPage.tsx

```diff
@@ -133,7 +137,7 @@ export const VmDetailsPage = ({
                     className="consoles-page-standalone"
                 >
                     <ConsoleCard
-                        state={consoleState}
+                        state={consoleCardState}
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2093-34c760d7-20250902-080839-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)